### PR TITLE
add singleton in glossary

### PR DESCRIPTION
**Pull request by anlexN**

### CODE REVIEW

**Review by br41nslug — CHANGES_REQUESTED** [review]
Thanks for the PR, just that little change and we should be all set 👍

**Comment by anlexN** [comment]
image

File: getting-started/glossary.md [hunk 1]
```diff
@@ -322,6 +322,16 @@ There is also a "Public" role that determines access for unauthenticated access.
 - [Configuring System Permissions](/configuration/users-roles-permissions#configure-system-permissions)
 - [Deleting a Role](/configuration/users-roles-permissions#deleting-a-role)
 
+## Singleton
+
+- **directus** - A collection that only contains one single item
```
Comment (br41nslug) [inline getting-started/glossary.md:327]:
```suggestion
- **Directus** - A collection that only contains one single item
```

File: getting-started/glossary.md [hunk 2]
```diff
@@ -322,6 +322,16 @@ There is also a "Public" role that determines access for unauthenticated access.
...
+- **design pattern** - classes which can be instantiated once, and can be accessed globally. This single instance can be shared throughout our application, which makes Singletons great for managing global state in an application
+
+### References
+
+- [`readSingleton`](https://github.com/directus/directus/blob/main/api/src/services/items.ts#L826)
+- [Singleton Pattern](https://www.patterns.dev/posts/singleton-pattern/)
```
Comment (br41nslug) [inline getting-started/glossary.md:333]:
```suggestion
- [**Design pattern**](https://www.patterns.dev/posts/singleton-pattern/) - Classes which can be instantiated once and can be accessed globally. This single instance can be shared throughout our application, which makes singletons great for managing global state in an application.
```
The link to github seems a bit much to me for the glossary and should really be over at the JS SDK section.

**Review by br41nslug — CHANGES_REQUESTED** [review]
Checked some grammar and capitalization if you could fix those that would be great 😄

**Review by br41nslug — APPROVED** [review]

**Comment by anlexN** [comment]
good job

**Pull request merged**